lib/node_modules/@stdlib/blas/ext/base/dnanasumors/test/test.ndarray.js

```diff
@@ -77,6 +77,18 @@ tape( 'the function calculates the sum of absolute values (ignoring NaN values)'
 	t.end();
 });
 
+tape( 'if provided a `stride` and `offset` parameter equal to `0` and the first element is NaN, the function returns `0`', function test( t ) {
```

Comment:
@Neerajpathak07 Also, I don't think the offset parameter is actually relevant here. Offset can be any value, not just `0`, and when the stride is `0` and the first **indexed** element is `NaN`, the function should always return `0`.